prod.yaml.jinja

```diff
@@ -94,7 +94,7 @@ services:
     labels:
       traefik.enable: "true"
       traefik.docker.network: "inverseproxy_shared"
-      {{- traefik2_labels.database(
+      {{- traefik3_paths_labels.database(
```

Comment:
```suggestion
      {{- traefik3_labels_2.database(
```
I get this error because [import](https://github.com/Tecnativa/doodba-copier-template/blob/e653c6e0de09f00368dc85040df92d2852134982/prod.yaml.jinja#L7) is imported as alias `traefik3_labels_2`
![image](https://github.com/user-attachments/assets/305017d9-c457-4ac5-af63-a601ce242d58)

![image](https://github.com/user-attachments/assets/1098a16f-dd5f-4650-84cd-464e28fbb2cb)

Reply:
Thank you @celm1990